spyder/plugins/editor/widgets/tests/test_introspection.py

```diff
@@ -60,6 +60,7 @@ def test_space_completion(lsp_codeeditor, qtbot):
 
 @pytest.mark.slow
 @pytest.mark.first
+@flaky(max_runs=5)
```

Comment:
We should simply disable this test on CI. It checks a very simple thing but creates a lot of problems on the CI an is too flaky...

Reply:
It should be passing now without giving so much problems.